.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+  - "10.15.3"
+before_script:
+  - "cd review-notifications"
+  - "yarn"
+script:
+  - "yarn lint"
+  - "yarn test"
+notifications:
+  slack: innoprojectsummer2019:fdHkFUL7PGRdik1mA9CQBapb
```

Comment:
You should end each file with a new line. Here is an explanation why: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline/729795#729795